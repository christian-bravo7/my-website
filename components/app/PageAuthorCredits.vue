<template>
  <p class="z-50 text-gray-400 dark:text-gray-400 text-xs page-author-credits select-none">
    &copy; 2021 CHRISTIAN BRAVO
  </p>
</template>

<script lang="ts">
import { Vue, Component } from 'nuxt-property-decorator';

@Component
export default class PageAuthorCredits extends Vue {}
</script>

<style lang="scss" scoped>
.page-author-credits {
  position: fixed;
  right: rem(16);
  bottom: rem(16);
}
</style>
