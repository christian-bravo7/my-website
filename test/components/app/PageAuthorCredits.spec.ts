import PageAuthorCredits from '@/components/app/PageAuthorCredits.vue';
import { mount } from '@vue/test-utils';

describe('PageAuthorCredits component', () => {
  it('should render correclty', () => {
    const wrapper = mount(PageAuthorCredits);

    expect(wrapper.html()).toMatchSnapshot();
  });

  it('should display my page credits', () => {
    const wrapper = mount(PageAuthorCredits);

    const credits = wrapper.find('p');

    expect(credits.text()).toBe('Â© 2021 CHRISTIAN BRAVO');
  });
});
