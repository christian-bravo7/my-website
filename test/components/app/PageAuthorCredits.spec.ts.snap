// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageAuthorCredits component should render correclty 1`] = `
<p class="hidden md:block z-50 text-gray-400 dark:text-gray-400 text-xs page-author-credits select-none">
  Â© 2021 CHRISTIAN BRAVO
</p>
`;
